bgpd/bgp_zebra.c

```diff
@@ -2152,6 +2153,88 @@ static int iptable_notify_owner(int command, struct zclient *zclient,
 	return 0;
 }
 
+/* Process route notification messages from RIB */
+static int bgp_zebra_route_notify_owner(int command, struct zclient *zclient,
+					zebra_size_t length, vrf_id_t vrf_id)
+{
+	struct prefix p;
+	enum zapi_route_notify_owner note;
+	uint32_t table_id;
+	char buf[PREFIX_STRLEN];
+	afi_t afi;
+	safi_t safi;
+	struct bgp_node *rn = NULL;
+	struct bgp *bgp = NULL;
+	struct bgp_info *ri = NULL, *new_select = NULL;
+
+	if (!zapi_route_notify_decode(zclient->ibuf, &p, &table_id, &note,
+					&afi, &safi)) {
+		zlog_err("%s : error in msg decode", __PRETTY_FUNCTION__);
+		return -1;
+	}
+
+	bgp = bgp_lookup_by_vrf_id(vrf_id);
+	if (!bgp) {
+		flog_err(EC_BGP_INVALID_NODE,
+				"%s : bgp instance not found vrf %d",
+				__PRETTY_FUNCTION__, vrf_id);
```

Comment:
Indentation needs to be fixed, here and in most other `if` statements and continued function calls